app/controllers/transactions/merchants_controller.rb

```diff
@@ -11,9 +11,9 @@ def new
 
   def create
     if Current.family.transaction_merchants.create(merchant_params)
-      redirect_to transactions_merchants_path, notice: t(".success")
+      redirect_to transaction_merchants_path, notice: t(".success")
     else
-      render transactions_merchants_path, status: :unprocessable_entity, notice: t(".error")
+      render transaction_merchants_path, status: :unprocessable_entity, notice: t(".error")
```

Comment:
Know this was existing and not part of this issue, but if I'm not mistaken, `render transaction_merchants_path` is not valid, and `notice: t(".error")` will render a green notification with an error message (I must have missed this entirely in a prior review).

As a temp fix, you think something like this would work?

```rb
Current.family.transaction_merchants.create!(merchant_params)
redirect_to transaction_merchants_path, notice: t(".success")
```

Reply:
Yep, early on here, I'm generally thinking throwing 500s is a better strategy for us.  We _should_ have html validations and we can always come back later and handle specific errors with notifications if absolutely critical.

I think all the `if/else` stuff sort of organically started proliferating throughout the project (copy/paste, etc.), but I'll try to use this convention more moving forward.